style.css

```diff
@@ -112,6 +116,12 @@ button {
     height: 300px;
     width: 300px;
     margin: 35px;
+    box-shadow: 10px 10px 4px rgba(101, 110, 78, 0.616)
+}
+.container__img:hover {
```

Comment:
Try to follow the same pattern, i think it would be better `.container__image:hover` like classes above.

Reply:
it's ready

Reply:
more changes were added, I hope you like them.

Reply:
it's ready